template/src/ontology/Makefile.jinja2

```diff
@@ -82,12 +82,17 @@ RELEASE_ARTEFACTS = $(sort {% for release in project.release_artefacts %}{% if r
 all: all_odk
 
 .PHONY: all_odk
-all_odk: odkversion test all_assets
+all_odk: odkversion test all_assets {% if project.make_diff %}release_diff{% endif %}
```

Comment:
```suggestion
all_odk: odkversion test all_assets{% if project.make_diff %} release_diff{% endif %}
```

Reply:
Totally irrelevant change, but if you dont add it, you will get a spurious diff during update_repo (the added whitespace)

Reply:
Ok